LFS171x/sawtooth-material/sawtooth-tuna/client/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Can we add a comment to this file to specify the license using SPDX short identifiers?
<!-- SPDX-License-Identifier: Apache-2.0 -->

Reply:
Sure! At the very top? Good to add this comment to the Fabric index.html page as well?

Reply:
Yes, we definitely want this on the Fabric work too.